src/windows/wsladiag/wsladiag.cpp

```diff
@@ -64,6 +72,59 @@ int wsladiag_main(std::wstring_view commandLine)
         return 0;
     }
 
+    if (!debugShell.empty())
```

Comment:
Given that this argument is probably going to grow in complexity, I'd recommend creating a dedicated function to handle it